services/src/pro/contexts/mod.rs

```diff
@@ -137,6 +184,30 @@ where
         op
     }
 
+    async fn create_raster_query_processor(
+        &self,
+        operator: Box<dyn InitializedRasterOperator>,
+    ) -> geoengine_operators::util::Result<TypedRasterQueryProcessor> {
+        check_quota(&*self.user_db, &self.session).await?;
```

Comment:
It is checked here and injected into the StatisticsProcessor? Why is this necessary?

Reply:
it was leftover from the previous attempt. I removed it